src/vs/workbench/contrib/terminal/browser/terminalMenus.ts

```diff
@@ -614,6 +678,39 @@ export function setupTerminalMenus(): void {
 					},
 					group: ContextMenuGroup.Kill,
 				}
+			},
+			{
+				id: MenuId.TerminalTabContext,
+				item: {
+					command: {
+						id: TerminalCommandId.RevealInFinder,
+						title: localize('workbench.action.terminal.revealInFinder', "Reveal Current Working Directory in Finder"),
+						precondition: TerminalContextKeys.terminalHasCwdDetectionCapability
```

Comment:
Yes I agree we should hide it completely instead of disabling when we don't have this 👍